homework3/task02/tests/test_fast_calculate.py

```diff
@@ -0,0 +1,12 @@
+from time import perf_counter
+
+from project_fast_calculate.task2_fast_calculate import fast_calculate
+
+
+def test_fast_calculate():
+    time_start = perf_counter()
+    sum_result = fast_calculate()
+    time_end = perf_counter()
+
+    assert time_end - time_start < 60
```

Comment:
You don't need to test execution time here.

Reply:
Yes, you are right. Execution time here helps to choose needed amount of processes for calculation time should not be more than a minute. I removed test execution time.